app/src/main/java/org/thoughtcrime/securesms/preferences/ClearAllDataDialog.kt

```diff
@@ -93,10 +93,12 @@ class ClearAllDataDialog : DialogFragment() {
 
             when (step) {
                 Steps.INFO_PROMPT -> {
-                    binding.dialogDescriptionText.setText(R.string.dialog_clear_all_data_message)
+                    binding.dialogDescriptionText.setText(R.string.clearDataAllDescription)
                 }
                 Steps.NETWORK_PROMPT -> {
-                    binding.dialogDescriptionText.setText(R.string.dialog_clear_all_data_clear_device_and_network_confirmation)
+                    // ACL TODO - This was `dialog_clear_all_data_clear_device_and_network_confirmation` - need an equivalent?
+                    val txt = "Are you sure you want to delete your data from the network? If you continue you will not be able to restore your messages or contacts."
```

Comment:
Adding a note here to make sure this gets addressed

Reply:
Replaced with our newly-named string `clearDeviceAndNetworkConfirm`